int32 robotId
int32 currNode
bool isCurrVisited
int32[2] chosenEdge
